src/widget/wcuemenupopup.cpp

```diff
@@ -77,6 +133,28 @@ WCueMenuPopup::WCueMenuPopup(UserSettingsPointer pConfig, QWidget* parent)
             this,
             &WCueMenuPopup::slotSavedLoopCueManual);
 
+    m_pSavedJumpCue = std::make_unique<CueTypePushButton>(this);
+    m_pSavedJumpCue->setToolTip(
+            tr("Toggle this cue type between normal cue and saved jump, to \n"
+               "the current play position or the current beatjump size ") +
+            "\n\n" +
+            tr("Left-click: Toggle this cue type to saved beatjump, using \n"
+               "the current play position if not previous destination was known. \nIf "
```

Comment:
This doesn't work currently? It's using the beatjump size

Reply:
Yep, you're right - tooltip to be updated ASAP

Reply:
And what do you think about making the behavior match the tooltip?
(see my comment above)

Reply:
The current position already gets used on right-click, are you suggesting to swap the click actions? I think it would be great to keep the saved loop and saved jump consistent

Reply:
Yes, that's what I'm suggesting. But this aspect has no priority.

In my [comment above](https://github.com/mixxxdj/mixxx/pull/13216#issuecomment-2858722758) I tried to explain why I think the loop and jump cues are different and why consistency is no necessary IMO.

TL;DR loop cues set with the default size can easily be adjusted afterwards with the many loop controls we have
vs.
Jump cues can afterwards only be changed with the cue menu.

Reply:
Yeah that's a fair point. Do you want to change it or should I do so?

Reply:
Alright. Yes, please do so

Reply:
Updated.It should now only use the jump size if cuepos == playpos, otherwise use playpos if unset or swap jumppos<>>cuepos